alert-common/src/main/java/com/synopsys/integration/alert/common/action/api/JobResourceActions.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.action.api;
+
+import java.util.List;
+import java.util.UUID;
+
+import com.synopsys.integration.alert.common.action.ActionResponse;
+import com.synopsys.integration.alert.common.rest.model.JobFieldModel;
+
+public interface JobResourceActions {
```

Comment:
Should this extend `ResourceActions`?

Reply:
If I add another Generic to resource actions to define the type for the ids then yes it would be possible.  The id type for Jobs is UUID and other resources is a Long.  Hindsight being 2020 perhaps the ids should have always been a UUID but because we have two different ID types I didn't want to make it extend.

Reply:
Ok, I think this could be a future enhancement if we ever migrate to UUIDs across the board.